scipy/sparse/linalg/_norm.py

```diff
@@ -165,6 +176,13 @@ def norm(x, ord=None, axis=None):
             M = abs(x).max(axis=a)
         elif ord == -Inf:
             M = abs(x).min(axis=a)
+        elif ord == 'spec':
+            # Spectral norm
+            from scipy.sparse import csc_matrix 
+            from scipy.sparse.linalg import svds
+            x = csc_matrix(x, dtype=float)
+            u, s, vt = svds(x, k=1)
+            return s[0]
```

Comment:
The spectral norm algorithm is duplicated here, right? It might be more suitable to define a function and call it for the conditions where a spectral norm is to be used, as the duplication of code could be a maintenance burden.

Reply:
The `'spec'` alias should be removed.